libtenzir/builtins/operators/version.cpp

```diff
@@ -22,10 +22,10 @@ class version_operator final : public crtp_operator<version_operator> {
     auto builder = series_builder{};
     auto event = builder.record();
     event.field("version", tenzir::version::version);
-    event.field("major", std::stoull(tenzir::version::major));
-    event.field("minor", std::stoull(tenzir::version::minor));
-    event.field("patch", std::stoull(tenzir::version::patch));
-    event.field("tweak", std::stoull(tenzir::version::tweak));
+    event.field("build", tenzir::version::build_metadata);
```

Comment:
More of a nit for consistency, feel free to ignore this if you think it's not worth the effort.

```suggestion
    event.field("build", tenzir::version::build);
```

Reply:
I tried to implement this change, but it clashes with the `tenzir::version::build` namespace that is defined in `config.hpp.in`. I think it makes most sense to keep the longer name as it is used in https://semver.org/.